src/plugins/sortFriendRequests/index.tsx

```
@@ -48,28 +64,27 @@ export default definePlugin({
-    makeSubtext(text: string, user: User) {
-        const since = this.getSince(user);
-        return (
-            <Flex flexDirection="column" style={{ gap: 0, flexWrap: "wrap", lineHeight: "0.9rem" }}>
-                <span>{text}</span>
-                {!isNaN(since.getTime()) && <span>Received &mdash; {since.toDateString()}</span>}
-            </Flex>
-        );
-    }
+        return <div className={cl("wrapper")}>
+            {children}
+            {!isNaN(since.getTime()) && (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)